### ISSUE TITLE
Remote deployment fails for availability of ss / netstat

### ISSUE THREAD

**Issue by gbraad** [minishift/minishift#2534]
Remote deployment failed when `zsh` is default shell
```
&{[-F /dev/null -o ConnectionAttempts=3 -o ConnectTimeout=10 -o ControlMaster=no -o ControlPath=none -o LogLevel=quiet -o PasswordAuthentication=no -o ServerAliveInterval=60 -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null gbraad@10.0.75.101 -p 22] C:\Program Files\OpenSSH-Win64\ssh.exe <nil>}
About to run SSH command:
if ! type netstat 1>/dev/null; then ss -tln; else netstat -tln; fi
SSH cmd err, output: exit status 127: zsh:1: command not found: ss

Error running SSH command: ssh command error:
command : if ! type netstat 1>/dev/null; then ss -tln; else netstat -tln; fi
err     : exit status 127
output  : zsh:1: command not found: ss
```

if the check was correct, the dependency on `net-tools` would not be needed.

**Comment by gbraad** [comment]
Actually, the same error happens on bash:

```
if ! type netstat 1>/dev/null; then ss -tln; else netstat -tln; fi
SSH cmd err, output: exit status 127: bash: line 0: type: netstat: not found
bash: ss: command not found

Error running SSH command: ssh command error:
command : if ! type netstat 1>/dev/null; then ss -tln; else netstat -tln; fi
err     : exit status 127
output  : bash: line 0: type: netstat: not found
bash: ss: command not found
```

Note: command `netstat` and `ss` are both available

**Comment by gbraad** [comment]
This needs to be fixed for #2496
